autoscheduler/frontend/src/components/CourseSelectColumn/CourseSelectColumn.css

```diff
@@ -0,0 +1,36 @@
+.container {
+    width: 20%;
+    height: 100%;
+    padding-top: 1%;
+    padding-left: 1%;
+}
+
+/* Having this wrapper lets us fix the top & bottom of the .course-select-column of the page
+   so the picker scrolls "under" the button while still being within the bounds of the
+   page
+*/
+.column-wrapper {
+    position: relative;
+    width: 100%;
+    height: 93%;

```

Comment:
93% seems like an aribtrary number. I believe this may the reason why the Add Course Button gets cut off on my screen.

![image](https://user-images.githubusercontent.com/10082177/77501842-257dc200-6e27-11ea-9c49-32bb8062247b.png)

Try replacing it with `calc(100% - 52px)`. This gives a better result on my screen, and it should theoretically account for differences across all screen sizes, because it leaves excalty 52px for the button regardless of screen height.

Reply:
It's totally an arbitrary number lol. I figured I had to do some kind of calculation but didn't know where to start really.

Setting that it to what you suggested made it touch the bottom for both 1080p & 1440p (1440p's even lower down than 1080)

Reply:
To be honest I have no idea what the best way to remedy this, as every solution I have tried gives a different distance from the Add Course button and the button between 1080p and 1440p. The only thing I can think of is that we could fix the bottom of the Add Course button to a certain amount of pixels above the bottom (similar to what we do with the cards column), then there may be a marginal difference between the heights of the column between screens, but as long as it falls at or below the top of the Add Course button it shouldn't matter? But I really have no idea how well that would work.

Reply:
Check out #171 for my proposed fix to this

Reply:
Merged #171, resolving